pkg/nuclide-external-interfaces/1.0/atom.js

```diff
@@ -900,6 +904,7 @@ declare class atom$TextEditorComponent {
     linesClientRect?: {top: number, left: number, bottom: number, right: number},
   ): {top: number, left: number, bottom: number, right: number},
   invalidateBlockDecorationDimensions(decoration: atom$Decoration): void,
+  getVisibleRowRange: [?number, ?number],
```

Comment:
Well, we're already pretty far into undocumented territory here so I'll allow it